packages/openneuro-indexer/src/indexQuery.ts

```diff
@@ -50,9 +50,26 @@ export const indexQuery = gql`
                 sex
                 age
               }
+              subjects
             }
             readme
           }
+          draft {
+            description {
+              Name
+            }
+            issues {
+              severity
+            }
+          }
```

Comment:
Do we actually need to index the draft? This is a bit expensive and really data should be snapshotted before it's searchable.